pkg/controller/build/helpers.go

```diff
@@ -331,11 +332,11 @@ func GetImageBuilderType(cm *corev1.ConfigMap) (string, error) {
 		return defaultBuilder, nil
 	}
 
-	validImageBuilderTypes := sets.NewString(OpenshiftImageBuilder, CustomPodImageBuilder)
+	validImageBuilderTypes := sets.NewString(string(OpenshiftImageBuilder), string(CustomPodImageBuilder))
 	if !validImageBuilderTypes.Has(configMapImageBuilder) {
 		return "", fmt.Errorf("invalid image builder type %q, valid types: %v", configMapImageBuilder, validImageBuilderTypes.List())
```

Comment:
suggestion: Since it looks like we perform this check here and elsewhere, it might make sense to consolidate this into a single function:

```golang
def validateImageBuilderType(imgBuilder ImageBuilderType) error {
    validImageBuilderTypes := sets.New[ImageBuilderType](OpenshiftImageBuilder, CustomPodImageBuilder)

    if !validImageBuilderTypes.Has(configMapImageBuilder) {
        return fmt.Errorf("invalid image builder type %q, valid types: %v", imgBuilderType, validImageBuilderTypes.List())
    }

    return nil
}
```